server/protocols/jmap-integration-testing/jmap-integration-testing-common/src/test/java/org/apache/james/jmap/methods/integration/GetMailboxesMethodTest.java

```diff
@@ -764,4 +764,38 @@ public void getMailboxesShouldReturnPartiallyAllowedMayPropertiesWhenDelegated()
             .body(FIRST_MAILBOX + ".mayDelete", equalTo(false))
             .body(FIRST_MAILBOX + ".mayRename", equalTo(false));
     }
+
+    @Test
+    public void getMailboxesShouldReturnUnlimitedQuotasForInboxByDefault() throws Exception {
+        String mailboxId = mailboxProbe.createMailbox(MailboxConstants.USER_NAMESPACE, alice, DefaultMailboxes.INBOX).serialize();
+
+        given()
+            .header("Authorization", accessToken.serialize())
+            .body("[[\"getMailboxes\", {\"ids\": [\"" + mailboxId + "\"]}, \"#0\"]]")
+        .when()
+            .post("/jmap")
+        .then()
+            .statusCode(200)
+            .body(NAME, equalTo("mailboxes"))
+            .body(ARGUMENTS + ".list", hasSize(1))
+            .body(FIRST_MAILBOX + ".quotas['key']['STORAGE'].max", nullValue())
+            .body(FIRST_MAILBOX + ".quotas['key']['MESSAGE'].max", nullValue());
```

Comment:
I'm a bit surprised by this `['key']` stuff... 

Could you confirm how the quotas looks like in plain JSON ?

(Do we have the spec somewhere?)

From what I read I understand:

```
...
"quotas": [
   {
        "key": "STORAGE"
         "max": null,
         "used": 0
    },
   {
        "key": "MESSAGE"
         "max": null,
         "used": 0
    }
],
...
```

Reply:
yes the spec is on the JIRA. I can confirm that the JSON looks fine. This 'key' is temporary, it will be replaced by the quotaroot is a following commit.